python/ray/workflow/tests/test_logging.py

```diff
@@ -0,0 +1,125 @@
+from ray._private.test_utils import run_string_as_driver_nonblocking
+
+
+def test_basic_workflow_logs(tmp_path):
```

Comment:
@lchu-ibm  it's not testing it correctly since it's like you start the workflow in the same driver.


```
def test_basic_workflow_logs(workflow_start_regular, tmp_path):
    script = f"""
        ray.init(address='auto')
        workflow.init("{workflow_start_regular}")
       ....
      ....
   """
   # check log
```

Reply:
@iycheng ah right!  fixing it now!